api/Console.json

```diff
@@ -1353,19 +1386,19 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/timestamp",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "14"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "39"
```

Comment:
I've confirmed with software.hixie.ch/utilities/js/live-dom-viewer/saved/8415 that `console.timeStamp` was introduced in Firefox 39. However, there's a capitalization mismatch here. Can you file an issue to follow up on this?

Reply:
Ah, you'd already sent https://github.com/mdn/browser-compat-data/pull/6650. Great!